mediawiki/docker-compose.yml

```diff
@@ -42,7 +42,7 @@ services:
     environment:
     - PORT=8005
     - PARSOID_DOMAIN=parsoid
-    - MW_API_ENDPOINT=http://127.0.0.1:8004/w/api.php
+    - MW_API_ENDPOINT=http://127.0.0.1:8004/api.php
```

Comment:
Maybe you should do the same edit here: https://github.com/tolstoyevsky/mmb/blob/master/mediawiki/docker-compose-armhf.yml#L44